arch/arm/core/aarch32/cortex_m/pm_s2ram.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2022, Carlo Caione <ccaione@baylibre.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/arch/cpu.h>
+#include <zephyr/init.h>
+
+#include <zephyr/arch/common/pm_s2ram.h>
```

Comment:
nit: please revisit includes here, e.g. `init.h` is not needed.

Reply:
I need that for `__noinit`

Reply:
`__noinit` comes from `include/zephyr/linker/section_tags.h`

Reply:
yeah, that is included by the include-chain by `init.h` :) to me it makes more sense to have that included through `init.h` than `section_tags.h`.

Reply:
done ;)